### PR TITLE
Add Policheck suppression file to Arcade SDL tool execution

### PR DESCRIPTION
**Pull request by MattGal**
tools/Licenses/MicrosoftDotNetLibrary.txt is lawyer-approved and needs to be ignored in scans.

### CODE REVIEW

**Review by riarenas — APPROVED** [review]
LGTM. Maybe run an official build with these changes to make sure the exclusion file does work?

File: azure-pipelines.yml [hunk 1]
```diff
@@ -265,3 +265,4 @@ stages:
         -TsaRepositoryName "Arcade"
         -TsaCodebaseName "Arcade"
         -TsaPublish $True'
+        -PoliCheckAdditionalRunConfigParams @("UserExclusionPath < $(Build.SourcesDirectory)/s/eng/PoliCheckExclusions.xml")
```
Comment (riarenas) [inline azure-pipelines.yml:268]: It looks like `Build.SourcesDirectory` already includes `s`


Build.SourcesDirectory | The local path on the agent where your source code files are downloaded. For example: c:\agent_work\1\s
-- | --




https://docs.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml#build-variables-devops-services

Comment (MattGal) [reply azure-pipelines.yml:268]: I did think it was weird, but I copied it from https://github.com/dotnet/aspnetcore/blob/main/eng/sdl-tsa-vars.config .  I will fix it up and try it on the official pipeline, and if that works will mention it to the ASP folks.

**Comment by MattGal** [comment]
Issues ostensibly resolved, https://dev.azure.com/dnceng/internal/_build/results?buildId=974247&view=logs&j=7d9eef18-6720-5c1f-4d30-89d7b76728e9&t=c49a855a-1eb9-5019-62bf-639a659fe361 will show green if the license file scan suppression works.

**Pull request merged**